plugins/restful/RestfulDataProviderEFQ.php

```diff
@@ -140,7 +140,9 @@ protected function queryForListSort(\EntityFieldQuery $query) {
 
     foreach ($sorts as $public_field_name => $direction) {
       // Determine if sorting is by field or property.
-      $property_name = $public_fields[$public_field_name]['property'];
+      if (!$property_name = $public_fields[$public_field_name]['property']) {
```

Comment:
I think we should use ``\RestfulDataProviderEFQ::getColumnFromProperty`` here (and maybe improve it to return FALSE of no ``schema field`` found) - as EMW allows defining "computed" properties that have no real schema field.